app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+{.alert .alert-error}
+
+## Styling at the document level
+	
+Browsers that implement the current Shadow DOM v1 specifications will automatically encapsulate styles, scoping them to the elements in which they were defined.
+
+Some browsers have not implemented the Shadow DOM v1 specifications. To make sure your apps and elements display correctly in these browsers, you'll need to use `custom-style` to ensure that styling information doesn't "leak" into the local DOM of your elements.
+
+The `custom-style` extension enables a set of polyfills that ensure that styles in your apps and elements behave as you would expect from the Shadow DOM v1 specifications, even in browsers that don't implement these specifications.
+
+`custom-style` is an extension of the native `<style>` element. To ensure that your styles behave according to the Shadow DOM v1 specifications in all browsers, use `custom-style` when you define *document-level* styles.
+
+*Note: You should only use `custom-style` to define styles for the main document. To define styles for an element's local DOM, just use a `<style>` block.*
+
+### Examples
+
+In the first code sample, the style for the `p` element “leaks” into Paragraph B in browsers that haven’t implemented the Shadow DOM v1 specs. In the second code sample, the developer has used `custom-style` to wrap the style block, preventing this leak.
+
+[See it on Plunker](http://plnkr.co/edit/0o1zuMHgmt4novf2DS8z?p=preview)
+
+```html
+  <style>
+    p {
+      color: red;
+    }
+  </style>
+  <p>Paragraph A: I am in the main DOM. I am red.</p>
+  <dom-module id="x-foo">
+    <template>
+      <p>Paragraph B: I am in the local DOM of x-foo. 
+         If your browser implements the Shadow DOM v1 specs, 
+         I am black; otherwise, I'm red.
+      </p>
+    </template>
+  </dom-module>
+  <x-foo></x-foo>   
+```	
+
+[See it on Plunker](http://plnkr.co/edit/yiD9XWPHaMjHaGGwu4V9?p=preview)
+```html
+  <custom-style>
+  <style>
+      p {
+        color: red;
+      }
+    </style>
+  </custom-style>
+  <p>Paragraph A: I am in the main DOM. I am red.</p>
+  <dom-module id="x-foo">
+    <template>
+      <p>Paragraph B: I am in the local DOM of x-foo. I am black on all browsers.</p>
+    </template>
+  </dom-module>
+  <x-foo></x-foo>   
+```
+  
+### Syntax and compatibility
+
+#### Polymer 2.x
+```html
+<custom-style>
+  <style>
+    p {
+		...
+    }
+  </style>
+</custom-style>
+````
+#### Compatibility between 1.x and 2.x
+```html
+<custom-style>
+  <style is="custom-style">
+    p {
+		...
+    }	
+  </style>
+</custom-style>
+``````
+#### Polymer 1.x
+```html
+<style is="custom-style">
+  p {
+	  ...
+  }
+</style>
+```
+
+### `custom-style` and custom properties
+
+In a DOM module template, you can define custom properties to be used by Polymer's shim for cross-scope styling. Then, use an `include` attribute in a `style` tag to share that style data between multiple custom-style elements. 
```

Comment:
This makes it sound like you need to use a style module to define custom properties, but that's not the case. They just need to be in a custom-style.

Maybe, since this doesn't introduce custom properties, this section should be in the custom properties chapter, and be called something like, "Setting custom properties at the document level" with a quick x-ref back here?